pkg/Microsoft.Private.CoreFx.NETCoreApp/netcoreapp.rids.props

```diff
@@ -1,85 +1,26 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Choose>
-    <When Condition="'$(PackageRID)' != ''" />
-    <When Condition="'$(_runtimeOSFamily)' == 'win'">
-      <PropertyGroup>
-        <PackageRID Condition="'$(ArchGroup)' == 'x86' OR '$(ArchGroup)' == 'x64'">win7-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(ArchGroup)' == 'arm'">win8-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(ArchGroup)' == 'arm64'">win10-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">win-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(_runtimeOSFamily)' == 'osx'">
-      <PropertyGroup>
-        <PackageRID>osx.10.12-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">osx-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <When Condition="'$(_runtimeOSFamily)' == 'rhel'">
-      <PropertyGroup>
-        <PackageRID>rhel.7-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">linux-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </When>
-    <Otherwise>
-      <PropertyGroup>
-        <PackageRID>$(RuntimeOS)-$(ArchGroup)</PackageRID>
-        <PackageRID Condition="'$(PortableBuild)' == 'true'">linux-$(ArchGroup)</PackageRID>
-      </PropertyGroup>
-    </Otherwise>
-  </Choose>
+  <PropertyGroup>
+    <PackageRID>win-$(ArchGroup)</PackageRID>
+    <PackageRID Condition="'$(OSGroup)' == 'OSX'">osx-$(ArchGroup)</PackageRID>
+    <PackageRID Condition="'$(OSGroup)' == 'Linux'">linux-$(ArchGroup)</PackageRID>
+  </PropertyGroup>
 
   <ItemGroup>
-    <OfficialBuildRID Include="alpine.3.4.3-x64" />
-    <OfficialBuildRID Include="debian.8-armel">
-      <Platform>armel</Platform>
-    </OfficialBuildRID>
-    <OfficialBuildRID Include="debian.8-x64" />
-    <OfficialBuildRID Include="fedora.24-x64" />
-    <OfficialBuildRID Include="linux-x64" />
-    <OfficialBuildRID Include="opensuse.13.2-x64" />
-    <OfficialBuildRID Include="opensuse.42.1-x64" />
-    <OfficialBuildRID Include="osx.10.12-x64" />
-    <OfficialBuildRID Include="osx-x64" />
-    <OfficialBuildRID Include="rhel.7-x64" />
-    <OfficialBuildRID Include="tizen.4.0.0-armel">
```

Comment:
As I know, currently we do not generate `linux-armel` packages so tizen.4.0.0-armel and debian-armel should be kept. And even though we add `linux-armel` packages, it may not be compotible in tizen by different library versions.